src/test/resources/inventory_view/instance_ids_underlying_srs_records_with_cyrillic.json

```diff
@@ -0,0 +1 @@
+{  "instanceId": "ebbb759a-dd08-4bf8-b3c3-3d75b2190c41", "source": "FOLIO", "updatedDate": "2020-06-15T11:07:48.563Z",  "deleted": "false",  "suppressFromDiscovery": "false"}{  "instanceId": "ac9ac919-a35e-4a2d-b0a7-8cb5a63ea20d", "source": "FOLIO", "updatedDate": "2020-06-15T11:07:48.732Z",  "deleted": "false",  "suppressFromDiscovery": "false"}
```

Comment:
is there supposed to be a newline between the two records?

Reply:
or a comma?  both?

Reply:
never mind, I guess not.  The API docs aren't very clear about this, but I tried it and this looks right.